playbooks/roles/airship-setup-deployer/tasks/main.yml

```diff
@@ -53,6 +53,7 @@
     - install
 
 - name: Ensure helm runs on localhost
+  become: yes
```

Comment:
why

Reply:
because someof us run the script on deployer, not as root user.